### PR TITLE
docs: Add link to the official announcement, links to examples, link to video tutorial

### PR DESCRIPTION
**Pull request by khorolets**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ NEAR Lake Framework is a small library companion to [NEAR Lake](https://github.c
 your own indexer that subscribes to the stream of blocks from the NEAR Lake data source and create your own logic to process
 the NEAR Protocol data.
 
+> [Official NEAR Lake Framework launch announcement](https://gov.near.org/t/announcement-near-lake-framework-brand-new-word-in-indexer-building-approach/17668) has been published on the NEAR Gov Forum
+> Greetings from the Data Platform Team! We are happy and proud to announce an MVP release of a brand new word in indexer building approach - NEAR Lake Framework.
```
Comment (frol) [inline README.md:8]: It is not readable:

image

File: README.md [hunk 2]
```diff
@@ -46,6 +49,15 @@ async fn handle_streamer_message(
 }
 ```
 
+Video tutorial:
+
+https://youtu.be/GsF7I93K-EQ
+
+### More examples
+
+- https://github.com/near/near-lake-raw-printer simple example of a data printer built on top of NEAR Lake Framework
```
Comment (frol) [inline README.md:58]: Do you have permissions to move printer to near-examples org? I think it makes sense to keep it there for now

**Review by frol — APPROVED** [review]
There are minor comments, so I am approving to unblock merging once those are addressed

**Review by telezhnaya — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (khorolets) [reply README.md]: did it

File: README.md (hunk 1 above)
Comment (khorolets) [reply README.md]: Ok, I do like the blockquote tag, but it's not so attractive in the dark mode :(

Comment (frol) [reply README.md]: I should have been more explicit. The fact that two paragraphs were joined into a single one without even a dot after "Forum" before "Greetings" makes it a confusing sentence. I am fine with the blockquote tag

**Pull request merged**